src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -1,22 +1,13 @@
-// Copyright 2019 Google LLC
```

Comment:
Could you potentially git rm this file if you push to master before me? Or pull from master before pushing if I push before you? Just wanna make sure we both have the same version of this file to avoid this happening again haha